scapy/contrib/lldp.uts

```diff
@@ -0,0 +1,168 @@
+from enum import test
+% LLDP test campaign
+
+#
+# execute test:
+# > test/run_tests -P "load_contrib('lldp')" -t scapy/contrib/lldp.uts
+#
+
++ Syntax check
+= Import the LLDP layer
+from scapy.contrib.lldp import *
```

Comment:
I copied this from other tests cause I was under the impression that this import test is done to check if the import fails cause something is wrong with the module path or with the modules syntax..

result of the first test keeping the include in the .uts (just added a X inside the lldp.py):
```
###(000)=[failed] Import the LLDP layer

>>> from scapy.contrib.lldp import *
Traceback (most recent call last):
  File "<input>", line 2, in <module>
  File "/home/hecke/projects/private/scapy/scapy/contrib/lldp.py", line 44, in <module>
    X
NameError: name 'X' is not defined
```
import test removed leads to regressions in functional tests:
```
###(000)=[failed] build basic LLDP frames

>>> frm = Ether(src='01:01:01:01:01:01', dst=LLDP_NEAREST_BRIDGE_MAC)/ \
...       LLDPDUChassisID(subtype=LLDPDUChassisID.SUBTYPE_MAC_ADDRESS, id=b'\x06\x05\x04\x03\x02\x01') / \
.......
...       LLDPDUEndOfLLDPDU()
Traceback (most recent call last):
  File "<input>", line 13, in <module>
NameError: name 'LLDP_NEAREST_BRIDGE_MAC' is not defined
```

so keeping the include test would give an early and clean indication that something is wrong with the module.

removed import [here](https://github.com/secdev/scapy/pull/701/commits/cf92af670c950b461c5c13cfd9ddad756a4ead8d) to follow the convention of the project.